src/main/java/org/gridsuite/directory/server/DirectoryService.java

```diff
@@ -122,7 +118,7 @@ private void emitDirectoryChanged(UUID directoryUuid, String userId, String erro
             userId,
             isPrivateForNotification(parentDirectoryUuid, elementAttributes.getAccessRights().isPrivate()),
             false,
-            NotificationType.UPDATE_DIRECTORY));
+            UPDATE_DIRECTORY));
```

Comment:
See previous comment about use of NotificationType.UPDATE_DIRECTORY

Reply:
Done